[Snowflake] Enterprise metrics tweak

shelldandy opened the pull request:
This makes the snowpark-metrics dashboard work when being imported straight from the plugin page in grafana.

- Format the JSON with prettier
- Removes `__inputs` field as they don't work with dashboards imported in this way
- Moves those variables to dashboard variables
- Downgrade minimum grafana version required from 11 to 9 to match the same min version supported by the plugin on grafana

https://github.com/snowflakedb/snowflake-telemetry-dashboard-templates/assets/7088511/b536ca97-ba6f-4436-8692-2b1520bb7c9a

Logs dashboard
![image](https://github.com/snowflakedb/snowflake-telemetry-dashboard-templates/assets/7088511/81ab127c-c4c1-4e99-8cb5-d976af61dec7)

shelldandy committed:
lint json

shelldandy committed:
removing __inputs

shelldandy committed:
properly point uids

shelldandy committed:
dowgrading required grafana version

this is to match the minimum supported version for the snowflake
datasource from grafana

shelldandy committed:
move memory_usage from __inputs to dashboard variable

shelldandy committed:
moving cpu_util from __inputs to dashboard variable

shelldandy commented:
@sfc-gh-sili please review this, added the commits in sequential order so they're a bit easier to follow, also cleared out extra stuff from my other PR that wasn't really needed to get it going

sfc-gh-vguttha commented on enterprise/snowpark-metrics.json:
```diff
@@ -398,12 +368,12 @@
           "datasource": {
             "type": "prometheus",
-            "uid": "${DS_GRAFANACLOUD-SNOWPARKTEST1-PROM}"
+            "uid": "prometheus"
```
Just for my learning, is there any reason this line change is needed?

shelldandy replied:
prometheus is part of every grafana install no need to point it to a specific id, the normal uid is enough

sfc-gh-vguttha commented on enterprise/snowpark-metrics.json:
```diff
@@ -669,49 +639,49 @@
-            "text": "${VAR_MEMORY_USAGE}",
-            "selected": false
+            "selected": true,
+            "text": "process.memory.usage",
```
do we want to hardcode?

sfc-gh-vguttha commented on enterprise/snowpark-metrics.json:
```diff
@@ -669,49 +639,49 @@
-            "value": "${VAR_MEMORY_USAGE}",
-            "text": "${VAR_MEMORY_USAGE}",
-            "selected": false
+            "selected": true,
```
we are updated selected value from false to true. Is this intended?

shelldandy replied:
it probably doesn't matter but currently as-is i get it working on fresh installs so might as well leave it there :)

shelldandy committed:
prettier on logs file

shelldandy committed:
removing unused elements

shelldandy committed:
removing unused __requires

shelldandy committed:
more proper pointing of ${datasource}

shelldandy committed:
removing __inputs from logs

dashboards imported from the plugin ui should use dashboard variables
instead of __inputs

shelldandy committed:
removing unused __elements and __requires from logs

shelldandy committed:
moving datasource to dashboard variable

shelldandy committed:
tweaking dashboard variables

shelldandy committed:
downgrading required grafana version

to match version supported by the plugin

shelldandy committed:
tweaking event_table options

shelldandy committed:
wider timegap

shelldandy committed:
adding uid to the dashboard

shelldandy committed:
hour unselected

shelldandy commented:
@sfc-gh-vguttha can we get this merged? its been approved for a while now
